tests/test_workflow_api.py

```diff
@@ -79,14 +82,208 @@ async def test_workflow_delete(
         }
 
         res = await client.post(
-            "api/v1/workflow/",
+            "/api/v1/workflow/",
             json=workflow,
         )
         wf_id = res.json()["id"]
 
-        res = await client.delete(f"api/v1/workflow/{wf_id}")
+        res = await client.delete(f"/api/v1/workflow/{wf_id}/")
         assert res.status_code == 204
 
         # TODO add tasks with API and test cascade delete
 
         assert not await db.get(Workflow, wf_id)
+
+
+async def test_get_workflow(
+    db, client, MockCurrentUser, project_factory, task_factory
+):
+    """
+    GIVEN a Workflow in the db
+    WHEN the get endpoint is called
+    THEN the Workflow is returned
+    """
+    async with MockCurrentUser(persist=True) as user:
+        project = await project_factory(user)
+        p_id = project.id
+        WF_ID = 1
+        workflow = {
+            "id": WF_ID,
+            "name": "My Workflow",
+            "project_id": p_id,
+            "task_list": [],
+        }
+        res = await client.post(
+            "/api/v1/workflow/",
+            json=workflow,
+        )
+
+        res = await client.get(f"/api/v1/workflow/{WF_ID}/")
+        assert res.status_code == 200
+        assert res.json() == workflow
+
+
+async def test_post_newtask(
+    db, client, MockCurrentUser, project_factory, task_factory
+):
+    """
+    GIVEN a Workflow with a list of Tasks
+    WHEN the POST endpoint is called
```

Comment:
specify the endpoint you're calling. e.g., "the endpoint to add a task to the task list is called"